### PR TITLE
Creating TypeDB Pre-Alpha Version [Only Standalone]

### PR DESCRIPTION
**Pull request by clintnetwork**
Building the Type-DB.Core Library in Pre-Alpha version that will works only in `Standalone` Mode.

Basic Example
-------
```csharp
using (var tdb = new TypeDB.Core(TypeDB.Mode.Standalone).Connect())
{
    using (var testdb = tdb.OpenDatabase("test", true))
    {
        testdb.Set<Player>("elena", new Player()
        {
            Name = "Elena",
            Age = 22
        });
    }
}
```

### CODE REVIEW

**Comment by clintnetwork** [comment]
Sucessfully Adding Travis

**Comment by clintnetwork** [comment]
@sondreb what do you think about this:

I would like to support some differents things like KeyValue, Events, Secrets Storing in Type-DB, so for this I have to possibilities:

When I open a database like this:
```csharp
var testdb = tdb.OpenDatabase("test", true)
```
After I can access to objects like this:
```csharp
testdb.Set<string>("test1", "319f27a8eb2c244846fcf6ee95fe7446");
```

or I need to make a sub namespace like this:
```csharp
testdb.KeyValue.Set<string>("test1", "319f27a8eb2c244846fcf6ee95fe7446");
```

We could have multiple namespaces such as Events, Secrets, KeyValue etc or we can put all methods in the same place.

Of course it would be cleaner to have many namespaces like this but if I ask you it's because firstly this involves the creation of several classes and secondly I'm not sure that Secrets or Events should be in the Database Object.

**To summarize, I am looking to create a relevant infrastructure that will allow us to add features in the future**
